java/src/main/java/com/cybersource/ws/client/Connection.java

```diff
@@ -66,6 +66,9 @@ protected Connection(MerchantConfig mc, DocumentBuilder builder,
     public static Connection getInstance(
             MerchantConfig mc, DocumentBuilder builder, LoggerWrapper logger) {
         if (mc.getUseHttpClient()) {
+            if(mc.getUsePoolingHttpClient()) {
```

Comment:
public static Connection getInstance(
            MerchantConfig mc, DocumentBuilder builder, LoggerWrapper logger) {
        if (mc.getUseHttpClient()) {
            return new HttpClientConnection(mc, builder, logger);
        }else if(mc.getUsePoolingHttpClient()) {
            return new PoolingHttpClientConnection(mc, builder, logger);
        } else {
            // HttpClient is not set in properties file then JDKHttpURLConnection class instance.
            return new JDKHttpURLConnection(mc, builder, logger);
        }
    }